Libs/MRML/Core/vtkDataFileFormatHelper.cxx

```diff
@@ -160,7 +160,7 @@ void vtkDataFileFormatHelper::PopulateITKSupportedWriteFileTypes()
         // Once we have the GetSupportedWriteGenericNames() ready, we will used that.
         // For now, just use the class name.
         ITKImageFileFormat structFileFormat =
-          {io->GetNameOfClass(), io->GetNameOfClass(), io->GetNameOfClass(),(*writeItr).c_str()};
+          {io->GetNameOfClass(), io->GetNameOfClass(), io->GetNameOfClass(),writeItr->c_str()};
```

Comment:
```suggestion
          {io->GetNameOfClass(), io->GetNameOfClass(), io->GetNameOfClass(), writeItr->c_str()};
```

Reply:
extra space after comment in new forced push.